digital-workspace/latest/process/starting.md

```diff
@@ -24,3 +24,19 @@ You can start a process in Alfresco Digital Workspace, with or without a file at
     Once the process starts you can view the running process by clicking **View running process** at the bottom of the window.
 
    > **Note:** You can also start a process by right clicking a file in your search results and selecting **Start Process**.
+
+## Badges
```

Comment:
I would call it `Part of running process badge` since it's about specific badge not about the mechanism in general